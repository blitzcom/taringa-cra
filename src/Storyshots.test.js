import initStoryshots from '@storybook/addon-storyshots'

Date.now = jest.fn(() => 1513924592705)

initStoryshots()
