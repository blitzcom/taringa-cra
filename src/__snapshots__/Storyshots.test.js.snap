// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Nav default 1`] = `
<div
  className="navbar navbar-dark Nav"
>
  <a
    className="navbar-brand font-weight-bold"
    href="/"
  >
    TARINGA!
  </a>
</div>
`;

exports[`Storyshots Story shout 1`] = `
<div
  className="card Story"
>
  <div
    className="Story-score-display"
  >
    <button
      className="Story-vote"
      onClick={[Function]}
    >
      <i
        className="fa fa-arrow-up"
      />
    </button>
    <div
      className="Story-score"
    >
      13.7k
    </div>
    <button
      className="Story-vote"
      onClick={[Function]}
    >
      <i
        className="fa fa-arrow-down"
      />
    </button>
  </div>
  <div
    className="Shout"
  >
    <a
      className="Shout-thumbnail"
      href="/"
    >
      <img
        alt="Thumbnail"
        src="https://k60.kn3.net/taringa/1/8/3/5/9/3/15/f3d3_14/61D.jpg"
      />
    </a>
    <div
      className="Shout-info"
    >
      <a
        className="Shout-title text-dark"
        href="/story/tienen-que-shoutear-minas-asi_z50xj"
      >
        Tienen que shoutear minas asi
      </a>
      <p
        className="Shout-meta"
      >
        Posteado por
        <a
          href="/user/F3D3_14"
        >
          F3D3_14
        </a>
        <time
          dateTime="2017-12-22T05:28:22.005Z"
          title="2017-12-22 05:28"
        >
          hace 1 hora
        </time>
      </p>
      <div
        className="Shout-actions"
      >
        <button
          className="btn btn-action"
          title="Expandir"
        >
          <i
            className="fa fa-expand"
          />
        </button>
        <button
          className="btn btn-action"
          title="Comentarios"
        >
          <i
            className="fa fa-comments"
          />
          8k
           comentarios
        </button>
        <button
          className="btn btn-action"
          title="Compartir"
        >
          <i
            className="fa fa-retweet"
          />
          4.5k
        </button>
      </div>
    </div>
  </div>
</div>
`;
